.circleci/config.yml

```diff
@@ -554,6 +587,26 @@ jobs:
             - "*.tar.gz"
       - *fail_notification
 
+  ubuntu_package:
+    executor: builder_container_1804
+    <<: *container_config
+    working_directory: ~/aeternity_deb
+    steps:
+      - checkout
+      - *build_ubuntu_package
+      - store_artifacts:
+          path: /tmp/package_tests/node1/log
```

Comment:
Who creates these folders? What are they meant to contain?

Reply:
Leftovers from the copy of `linux_package` from which the `ubuntu_package` rule was initially started. Probably I should remove them.

Reply:
yeah, please cleanup

Reply:
Addressed in 2cff31a